Jenkinsfile

```diff
@@ -105,15 +109,17 @@ pipeline {
           steps {
             script {
               def tasks = [:]
-                tasks["Kubernetes GKE, DAP"] = {
-                  sh "./bin/start --docker --dap --gke"
-                }
-                tasks["Openshift v3.11, DAP"] = {
-                  sh "./bin/start --docker --dap --oc311"
-                }
-                tasks["Openshift v4.5, DAP"] = {
-                  sh "./bin/start --docker --dap --oc45"
+              tasks["Kubernetes GKE, DAP"] = {
+                sh "./bin/start --docker --dap --gke"
+              }
+              tasks["Openshift (Current), DAP"] = {
```

Comment:
maybe we should add a test flow against `oldest` but comment it out for now? oldest is currently 4.3, and SP is not certified for 4.3